docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -1,37 +1,46 @@
-# Create a OpenStack cluster
+# Create an OpenStack cluster
 
-Install an OCP cluster running on VMs within a management OCP cluster
+This document explains how to create HostedClusters and Nodepools using the OpenStack platform.
+
+## Overview
+
+When you create a HostedCluster with the OpenStack platform, HyperShift will install the [OpenStack CAPI
+provider](https://github.com/kubernetes-sigs/cluster-api-provider-openstack) in the Hosted Control Plane (HCP) namespace.
+Upon scaling up a NodePool, a Machine will be created, and the CAPI provider will create the necessary resources in OpenStack.
 
 ## Limitations
 
-* The HyperShift Operator with OpenStack support is currently in development and is not intended for production use.
-* OpenStack CSI (Cinder and Manila) are not functional.
-* Operators running in the workload cluster (e.g. console) won't be operational on day 1 and a manual and documented
-  action is required to make them work on day 2.
+* Although the HyperShift Operator with OpenStack support is currently in development and is not intended for production use,
+  it is possible to create and manage clusters for development and testing purposes and it's expected to work as described in this document.
+* OpenStack CSI (Cinder and Manila) are not functional yet but are [expected](https://issues.redhat.com/browse/OSASINFRA-3536) to work in the 4.18 release and fully supported in future releases.
+* A few operators running in the workload cluster (e.g. console) won't be operational on day 1 because the floating IP used
+  for Ingress isn't automatically configured in the DNS. This is a manual and documented step that needs to be done after the
+  cluster is created. We are [working](https://issues.redhat.com/browse/OSASINFRA-3489) on allowing customers to pre-create the DNS records before the cluster is created by using
+  a pre-created floating IP.
 
 ## Prerequisites
 
 * Admin access to an OpenShift cluster (version 4.17+) specified by the `KUBECONFIG` environment variable.
+  This cluster is referred to as the Management OCP cluster.
 * The Management OCP cluster must be configured with OVNKubernetes as the default pod network CNI.
-* The OpenShift CLI (`oc`) or Kubernetes CLI (`kubectl`).
+* The OpenShift CLI (`oc`) or Kubernetes CLI (`kubectl`) must be installed.
+* The `hcp` CLI CLI must be installed.
```

Comment:
I'm confused... we've removed the `hypershift` binary as a requirement, but below we're still explaining how to install it, and using it to install the hypershift operator.

Is there an other way we can install the hypershift operator that does not involve the `hypershift` binary?

Reply:
Oops indeed. I'll fix it today.

Reply:
Done: https://github.com/openshift/hypershift/pull/5103